src/mmw/js/src/modeling/templates/scenarioToolbarTabContent.html

```diff
@@ -2,8 +2,12 @@
 
 <div id="modification-btn-wrapper"> <!-- Modifications Dropdown -->
     <div class="dropdown"> <!-- Dropdown Button -->
+        {% if shapes|length == 0 %}
```

Comment:
Since Nunjucks follows Python convention, consider using `if not shapes` instead of `if shapes|length == 0`